### DISCUSSION TITLE
php undefined variable(s)

### DISCUSSION THREAD

**Discussion by Rastalovich** [gboudreau/Greyhole#256] [Q&A] [answered]
Hi gboudreau,

Gh working well on a new install of your newest version.

I'm discovering the php page for admin/info and I notice no logs show on the page, and the php feedback shows: (I've launched the php -S as sudo and it does the same - but I haven't tried it running as root at the moment)

[Mon Jan  4 15:20:21 2021] PHP Notice:  Undefined variable: current_action in /usr/share/greyhole/web-app/init.inc.php on line 249[Mon Jan  4 15:20:21 2021] PHP Notice:  Undefined variable: current_action in /usr/share/greyhole/web-app/init.inc.php on line 249

Further down it complains when trash is empty: (no errors when there's something in the trash)

[Mon Jan  4 16:19:41 2021] Received POST ?ajax=get_trashman_content: []
[Mon Jan  4 16:19:41 2021] PHP Notice:  Undefined variable: output in /usr/share/greyhole/web-app/init.inc.php on line 401
[Mon Jan  4 16:19:41 2021] PHP Warning:  Invalid argument supplied for foreach() in /usr/share/greyhole/web-app/init.inc.php on line 401
[Mon Jan  4 16:19:41 2021] PHP Notice:  Undefined variable: num_rows in /usr/share/greyhole/web-app/init.inc.php on line 494
[Mon Jan  4 16:19:41 2021] PHP Notice:  Undefined variable: num_rows in /usr/share/greyhole/web-app/init.inc.php on line 494

Very impressed with this php, I do remember seeing the disk use graphic before :)

**Comment by gboudreau** [comment, marked as answer]
Thx. Will be fixed in next version.
